src/_GuestBook/RegularGuest.java

```diff
@@ -0,0 +1,19 @@
+package _GuestBook;
+
+public class RegularGuest implements Guest {
+	RegularGuest(String name){
+		this.name = name;
+	}
+	String name;
```

Comment:
Ordering files: maybe place all instance variable(s) at the top, before the constructor

Reply:
Also, I think it's safe to make the name variable private